nucypher/network/retrieval.py

```diff
@@ -121,7 +118,7 @@ def update_errors(self,
                       work_order: "RetrievalWorkOrder",
                       ursula_address: ChecksumAddress,
                       error_message: str):
-        for capsule in work_order.capsules:
+        for capsule in work_order.capsules():
```

Comment:
let's keep this as it was and use `@property` decorator

Reply:
Updated; it is only used within this module though.